app/src/main/java/com/example/socialapp/SharedPreferenceHelper.java

```diff
@@ -0,0 +1,21 @@
+package com.example.socialapp;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import androidx.preference.PreferenceManager;
+
+
+public class SharedPreferenceHelper{
+    static void setSharedPreferenceString(Context context, String key, String value){
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
+        SharedPreferences.Editor editor = preferences.edit();
+        editor.putString(key, value);
+        editor.apply();
+    }
+    static String getSharedPreferenceString(Context context, String key, String defValue){
```

Comment:
We we can have shorter name like `getString`.

Reply:
Implemented